### DISCUSSION TITLE
LionCalendar: Enable overriding previous & next button's icon template

### DISCUSSION THREAD

**Discussion by ghost** [ing-bank/lion#1953] [Open for contributions]
The LionCalendar component currently doesn't allow the changing of the previous and next button's icons without modifying the protected __renderPreviousButton and __renderNextButton methods.
My proposed solution would be to extract the arrows into two overridable methods called _previousIconTemplate and _nextIconTemplate respectively. This would decouple the icons from the underlying logic of the buttons.

**Comment by gerjanvangeest** [comment]
@barni52 sounds like a good idea. go ahead and create a PR i would say.

**Reply by ghost** [reply]
Yep, submitted a pull request here (https://github.com/ing-bank/lion/pull/1954) :)
